src/workloads/query/CsiFragmentedInsertsFlat.yml

```diff
@@ -0,0 +1,271 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-execution"
+Description: |
+  This workload compares performance of inserts into a collection with only the default _id index,
+  and in presence of either columnstore or wildcard indexes. It uses an artificial data set with
+  a wide overall schema and narrow individual objects to model fragmented access to CSI, which
+  clusters entries by path.
+
+Keywords:
+- columnstore
+- wildcard
+- insert
+
+AutoRun:
```

Comment:
I'm pretty unsure how this stuff works. This section means we don't have any new task associated with this workload, is that right? How will it show up in evergreen - both in your patch builds and once it's in the mainline? Do we need to worry about a 6.1 and 6.2 branch, or are those already disabled?

We have this open TODO, so I've been kinda ignoring this for the time being: https://jira.mongodb.org/browse/PERF-3094

Reply:
This looks right to me. The test is in a patch build.

Reply:
Thank you for the confirmation! I'll keep it as is.